app/templates/administrative-units/administrative-unit/governing-bodies/governing-body/mandatory/edit.hbs

```diff
@@ -148,21 +148,17 @@
                         />
                       </:content>
                     </Item>
-                    <Item
-                      @labelFor="contact-address-province"
-                      @errorMessage={{@model.address.error.province.validation}}
-                    >
+                  </:manual-address-input>
+                  <:common-input>
+                    <Item @labelFor="contact-address-province" @required={{true}}
```

Comment:
I don't think it needed to be required on the "position" pages. So the mandatory and minister creation and edit pages shouldn't make this required, but maybe I misunderstood in the meeting.